src/main/resources/js/sourcegraph-bitbucket.js

```diff
@@ -0,0 +1,23 @@
+AJS.toInit(() => {
+    // Fetch Sourcegraph URL on page load
+    AJS.$.ajax({
+        url: AJS.contextPath() + '/rest/sourcegraph-admin/1.0/',
+        dataType: 'json'
+    }).done(({ url }) => {
+        if (!url) {
+            console.log(`No Sourcegraph URL is set. To set a Sourcegraph URL, log in as a site admin and navigate to ${AJS.contextPath()}/plugins/servlet/sourcegraph.`)
+            return
+        }
+        // If a Sourcegraph URL is set,
+        // inject a <script> tag to fetch the main JS bundle
+        // from the Sourcegraph instance
+        window.SOURCEGRAPH_URL = url
+        window.localStorage.SOURCEGRAPH_URL = url
+        window.SOURCEGRAPH_INTEGRATION = 'bitbucket-integration'
+        var script = document.createElement('script')
+        script.type = 'text/javascript'
+        script.defer = true
```

Comment:
@lguychard will this cause a race condition with detecting whether a native integration is running from the browser extension?

Reply:
No. `<script>` tags with the `defer` attribute are executed after the document has been parsed, but before `DOMContentLoaded` fires. Browser extension content scripts are set to run at `document_end`, after `DOMContentLoaded` has fired.